src/Cohesity.Powershell/Scripts/View/New-CohesityView.ps1

```diff
@@ -0,0 +1,154 @@
+function New-CohesityView {
```

Comment:
This was C# based before correct ? have we made sure the ps based preserves the objects returned etc ?

Reply:
yeah, it was in c# before. It preserves the same response objects